tests/cisco_nxos/show_ip_interface_brief/cisco_nxos_show_interface_brief.parsed

```diff
@@ -0,0 +1,662 @@
+---
```

Comment:
This file should be removed. Appears to have the data from cisco_nxos_show_interface_brief